src/Sylius/Component/Product/tests/Generator/SlugGeneratorTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\Component\Product\Generator;
+
+use PHPUnit\Framework\TestCase;
+use Sylius\Component\Product\Generator\SlugGenerator;
+use Sylius\Component\Product\Generator\SlugGeneratorInterface;
+
+final class SlugGeneratorTest extends TestCase
+{
+    private SlugGenerator $slugGenerator;
+
+    protected function setUp(): void
+    {
+        $this->slugGenerator = new SlugGenerator();
+    }
+
+    public function testImplementsSlugGeneratorInterface(): void
+    {
+        self::assertInstanceOf(SlugGeneratorInterface::class, $this->slugGenerator);
+    }
+
+    public function testGeneratesSlugBasedOnGivenName(): void
+    {
+        $this->assertSame(
+            'cyclades',
+            $this->slugGenerator->generate('Cyclades')
+        );
```

Comment:
It can be in one line or should have `,` at the end:
```suggestion
        $this->assertSame(
            'cyclades',
            $this->slugGenerator->generate('Cyclades'),
        );
```